Examples/Detectors/MagneticField/src/FieldMapRootIo.cpp

```diff
@@ -102,7 +114,7 @@ ActsExamples::makeMagneticFieldMapXyzFromRoot(
     zPos.push_back(z);
     bField.push_back(Acts::Vector3(Bx, By, Bz));
   }
-  inputFile->Close();
+  delete inputFile;
```

Comment:
Does the destructor close the file correctly?

Reply:
Yes, it should. The `TTree` is owned by the `TFile`, so we don't delete the `TTree`.

As you know, there is a potential issue for `TFile`s owned by other objects if they are destroyed in the global destructor, since this can give a double delete if they are destroyed after ROOT's global destructor. That's not a problem here.

Reply:
Right. But I thought if we `TFile::Open` here, we must also `->Close()`. But ok if `delete` does the job I'm ok.